### PR TITLE
Update sktime links, add tuning grids for Dummy models

### PR DESCRIPTION
**Pull request by celestinoxp**
Some small updates

### CODE REVIEW

File: pycaret/containers/models/classification.py [hunk 1]
```diff
@@ -1495,8 +1495,12 @@ def __init__(self, experiment):
         from sklearn.dummy import DummyClassifier
 
         args = {"strategy": "prior", "random_state": experiment.seed}
+
         tune_args = {}
-        tune_grid = {}
+        tune_grid = {
```
Comment (ngupta23) [inline pycaret/containers/models/classification.py:1500]: @Yard1 I think you were going to make this change also. Please check if this matches your change. Also, similar change needs to be made for regression as well.

**Review by ngupta23 — APPROVED** [review]

**Comment by ngupta23** [comment]
ok, from the time series side. 

@Yard1 please review the change to grid. Thanks!

@celestinoxp Thanks for the PR. Please try to keep different issues in separate PRs. It makes it easier to get approvals :)

**Comment by celestinoxp** [comment]
> @celestinoxp Thanks for the PR. Please try to keep different issues in separate PRs. It makes it easier to get approvals :)

I dont´t know why vscode have created all inside same PR. Sorry. I will try configure VSCode.

I forgort the regression. I will keep in mind next times. Thanks

File: pycaret/containers/models/classification.py [hunk 2]
```diff
@@ -1495,8 +1495,12 @@ def __init__(self, experiment):
...
+            "strategy": ["most_frequent", "prior", "stratified", "uniform", "constant"]
```
Comment (Yard1) [inline pycaret/containers/models/classification.py:1501]:
```suggestion
            "strategy": ["most_frequent", "prior", "stratified", "uniform"]
```

File: pycaret/containers/models/regression.py [hunk 3]
```diff
@@ -1906,7 +1906,9 @@ def __init__(self, experiment):
             "random_state": experiment.seed,
             "n_jobs": 1 if experiment.gpu_param else None,
         }
-        tune_args = {}
+        tune_args = {
+            "strategy": ["most_frequent", "prior", "stratified", "uniform", "constant"]
+        }
```
Comment (Yard1) [inline pycaret/containers/models/regression.py:1911]:
```suggestion
        tune_args = {
            "strategy": ["mean", "median"]
        }
```

**Comment by Yard1** [comment]
Thanks!

**Pull request merged**